explain.rb

```diff
@@ -0,0 +1,6 @@
+date = Date.now
+puts date
+
+# puts date.strftime('%y/%m/%d')
+
+# puts date.strftime('%H/%M/%S')
```

Comment:
Nice job so far😉. As we have learned in the [ruby best practices requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md), Kindly take away every commented piece of code as this makes your codebase look messy and gives it a less professional look😉.

<img width="1083" alt="image" src="https://github.com/AliRezaBmeDu/school-library/assets/64643995/08c81e8b-315f-416c-8505-04ce3ce82fde">

Reply:
@Ibizugbe , The file was to test out some ruby commands. It does not participate in the project. Anyway, I am deleting it.